### DISCUSSION TITLE
CH2. 개략적인 규모 추정

### DISCUSSION THREAD

**Discussion by dojinyou** [Learning-Is-Vital-In-Development/23-7-SystemDesignInterview#12] [Q&A]
자유롭게 질문과 답변을 진행해봅니다!

**Comment by dojinyou** [comment]
참고하면 좋은 자료(by 건창님)
https://www.youtube.com/watch?v=rxurfKT2lD8&t=575s

**Reply by dojinyou** [reply]
@this-is-spear

**Reply by this-is-spear** [reply]
USE 방법론

성능 개선 지점을 판단해 빠르게 개선하기 위한 프로세스입니다. 어떤 단어의 줄임말인지 이해하고 접근하면 쉬울 것 같아 작성해둡니다!


- Utilization : 얼만큼 자원을 썼는지
- Saturation : 얼마나 많은 부하가 몰리는지
- Error : 에러가 발생했는지

따로 참고자료는 아래 있습니다. (USE 방법론과 RED 방법론에 관한 글입니다.)

[참고자료](https://www.pusnow.com/note/use-method-and-red-method/)

**Comment by this-is-spear** [comment]
[HDD와 SDD 관점에서 디스크 탐색과 순차적 탐색 비교](https://github.com/this-is-spear/system-design-interview/wiki/%EB%94%94%EC%8A%A4%ED%81%AC-%ED%83%90%EC%83%89%EC%9D%84-%ED%94%BC%ED%95%98%EB%9D%BC%EB%8A%94-%EB%A7%90%EC%9D%98-%EC%9D%98%EB%AF%B8%EB%A5%BC-%EB%AC%BC%EB%A6%AC%EC%A0%81%EC%9D%B8-%EB%A6%AC%EC%86%8C%EC%8A%A4-%EA%B4%80%EC%A0%90%EC%97%90%EC%84%9C-%EC%84%A4%EB%AA%85%ED%95%B4%EB%B3%B4%EA%B2%A0%EC%8A%B5%EB%8B%88%EB%8B%A4.) 글 공유드립니다!
